File: 프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.java
class Solution {
    public String solution(int[] food) {
        String str = "";
        for(int i = 1; i < food.length; i++) {
            int food_count = food[i] % 2 == 0 ? food[i] : food[i] -1;
            str += (i + "").repeat(food_count / 2);
        }
        StringBuffer sb = new StringBuffer(str);
        return sb.toString() + "0" + sb.reverse().toString();
    }
}

File: 프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.js
function solution(food) {
    let result = [];
    let food_count = food.slice(1);
    food_count.forEach((x, i) => {
        if(x % 2 !== 0) {
            x--;
        }
        let str = ((i + 1) + '').repeat(x / 2);
        result.push(str);
    });
    return `${result.join('')}0${result.reverse().join('')}`;
}